exporter/opentelemetry-exporter-prometheus-remote-write/src/opentelemetry/exporter/prometheus_remote_write/__init__.py

```diff
@@ -66,6 +66,7 @@ class PrometheusRemoteWriteMetricsExporter(MetricExporter):
         timeout: timeout for remote write requests in seconds, defaults to 30 (Optional)
         proxies: dict mapping request proxy protocols to proxy urls (Optional)
         tls_config: configuration for remote write TLS settings (Optional)
+        auth: auth tuple or callable to enable Basic/Digest/Custom HTTP Auth (Optional)
```

Comment:
The callable should be compatible with requests though, so it should take a requests' Request as parameter and set whatever header is required. The problem with that though is that it'll leak the implementation. So maybe is it better to expand the documentation of the exporter to add an example on how to add authentication using just headers instead?